web/controllers/class_controller.ex

```diff
@@ -18,13 +19,20 @@ defmodule CoursePlanner.ClassController do
   def create(conn, %{"class" => class_params}) do
     changeset = Class.changeset(%Class{}, class_params, :create)
 
-    case Repo.insert(changeset) do
+    multi =
+      Multi.new
+      |> Multi.insert(:class, changeset)
```

Comment:
We should have one transaction to create all attendances together, but I am not sure about the class and the attendances being created in the same transaction

What can the user do if the transaction fails because of the attendances?